index.bs

```diff
@@ -776,19 +789,19 @@ interface XRReferenceSpace : XRSpace {
 };
 </pre>
 
-Each {{XRReferenceSpace}} has a <dfn for="XRReferenceSpace">type</dfn>, which is an {{XRReferenceSpaceType}}.
+The [=XRSpace/internal type=] of an {{XRReferenceSpace}} is one of the variants of {{XRReferenceSpaceType}}, chosen during construction..
 
 An {{XRReferenceSpace}} is most frequently obtained by calling {{XRSession/requestReferenceSpace()}}, which creates an instance of an {{XRReferenceSpace}} or an interface extending it, determined by the {{XRReferenceSpaceType}} enum value passed into the call. The type indicates the tracking behavior that the reference space will exhibit:
 
-  - Passing a type of <dfn enum-value for="XRReferenceSpaceType">identity</dfn> creates an {{XRReferenceSpace}} instance. It represents a reference space where the [=viewer=] is always at the [=native origin=]. Only the {{XRSession/viewerSpace}} and {{XRInputSource/targetRaySpace}}'s of {{XRInputSource}}'s with an {{XRInputSource/targetRayMode}} of {{XRTargetRayMode/screen}} can be spatially related to an {{identity}} reference space.
+  - Passing a type of <dfn enum-value for="XRReferenceSpaceType">identity</dfn> creates an {{XRReferenceSpace}} instance. It represents a reference space where the [=viewer=] is always at the [=native origin=]. Only the {{XRSession/viewerSpace}} and {{XRInputSource/targetRaySpace}}'s of {{XRInputSource}}'s with an {{XRInputSource/targetRayMode}} of {{XRTargetRayMode/screen}} can be spatially related to an {{identity}} reference space. Its [=native origin=] is the position and orientation of the [=viewer=].
```

Comment:
> It represents a reference space where the [=viewer=] is always at the [=native origin=].

> Its [=native origin=] is the position and orientation of the [=viewer=].

These lines seem to both be stating the same thing. Can we condense it down to a single line?

Reply:
I read the first one as saying what it's for, and the second one as defining how it works internally. But yeah I can condense it.